pkg/acsengine/defaults.go

```diff
@@ -83,6 +83,14 @@ var (
 		ImageVersion:   "latest",
 	}
 
+	//DefaultOpenShift39RHELImageConfig is the OpenShift on RHEL distribution.
+	DefaultOpenShift39RHELImageConfig = AzureOSImageConfig{
+		ImageOffer:     "acsengine-preview",
```

Comment:
do we want our default to be origin?  or do we prefer rhel here?

Reply:
This is not actually a default that is set in the ARM template and the variable name is probably confusing but I just followed the naming convention used in the rest of the image config vars in this file.